src/fabio_rodrigues_ex/ex02/file_stats.py

```diff
@@ -0,0 +1,30 @@
+def char_counts(textfilename):
+    """
+    :param:
+    1st. Opens the file with the given filename using encoding utf-8.
+    2nd. Reads the entire file content into a single string.
+    3rd. Counts how often each character code (0–255) occurs in the string.
+    :return:
+    The result as a list or tuple, where result[i] gives the number of occurrences of character code i.
```

Comment:
Line's too long here, over 79 characters. This PEP8 violation (I noticed that by email twice).

Reply:
> Nice code. Unfortunately PEP8 is violated, which causes an automatic 'fail'. Please remember the code line next time before creating the pull request.

Hi huynhngoc! 

I think this is not fair to fail me because of characters in a comment without give me a second chance to fix it... 

I have done all the procedure from the youtube video made by the professor, included the 79 character violation. I have checked the sign in the top right of my pychamp and everyting was OK... I do not know what happened...

But is it realy unfair to fail me because of that without give me a chance to fix.. This is a really small fix that does not deserve a fail... And I HAVE DONE THE PROCEDURE FROM THE YOUTUBE VIDEO... Something went wrong and I will revise it...

What should I do now? Can I merge the files?